src/Utility /label.js

```diff
@@ -0,0 +1,2 @@
+export const appName = "YouTube Video Search"
```

Comment:
@RajanTank create JSON obj and then export it
const label = {
    loginSuccess: "login successfully!",
    loginFail: "Invalid email and password.",
   }